scripts/azureml-assets/azureml/assets/release/e2e_test.py

```diff
@@ -17,11 +17,15 @@
     parser.add_argument("-s", "--subscription", required=True, type=str, help="Subscription ID")
     parser.add_argument("-r", "--resource-group", required=True, type=str, help="Resource group name")
     parser.add_argument("-w", "--workspace-name", required=True, type=str, help="Workspace name")
+    parser.add_argument("-c", "--coverage-report", required=False, type=Path, help="Path of coverage report yaml")
+    parser.add_argument("-v", "--version-suffix", required=False, type=str, help="version-suffix")
```

Comment:
nit: Provide a little more help text than the name of the arg.

Reply:
done